templates/kubeflow/install.html

```diff
@@ -13,7 +13,16 @@ <h1>Install Kubeflow on Ubuntu</h1>
       <p>Create and train machine learning models on your laptop, in your data center, or in the cloud.</p>
     </div>
     <div class="col-5 u-align--center u-hide--small">
-      <img src="https://assets.ubuntu.com/v1/a1317807-Install+Kubeflow.svg" width="290" alt="Install Kubernetes" />
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/a1317807-Install+Kubeflow.svg",
+          alt="Install Kubernetes",
+          height="200",
+          width="290",
+          hi_def=True,
+          loading="lazy",
```

Comment:
First image in header should not be lazy...

```suggestion
          loading="auto",
```